### PR TITLE
OCPQE-24207 add release gate test job for arm64

### PR DESCRIPTION
**Pull request by wangke19**
Depends on the release repo pr https://github.com/openshift/release-tests/pull/236

### CODE REVIEW

File: ci-operator/config/openshift/release-tests/openshift-release-tests-master.yaml [hunk 1]
```diff
@@ -62,6 +62,13 @@ tests:
   steps:
     test:
     - ref: release-qe-tests
+- as: qe-arm-release-gate-test
```
Comment (rioliu-rh) [inline ci-operator/config/openshift/release-tests/openshift-release-tests-master.yaml:65]: we should align with the job name conversion
e.g. `qe-release-gate-test-$arch`

**Comment by rioliu-rh** [comment]
you need to run `make update` to generate prow job definition and reformat your config file

File: ci-operator/config/openshift/release-tests/openshift-release-tests-master.yaml [hunk 2]
```diff
@@ -62,6 +62,13 @@ tests:
...
+  cron: 15 * * * *
```
Comment (rioliu-rh) [inline ci-operator/config/openshift/release-tests/openshift-release-tests-master.yaml:66]: I think we'd better to make the schedule different with amd64 job, e.g. every 90 mins

**Comment by rioliu-rh** [comment]
/pj-rehearse periodic-ci-openshift-release-tests-master-qe-release-gate-test-arm64

File: ci-operator/config/openshift/release-tests/openshift-release-tests-master.yaml (hunk 2 above)
Comment (wangke19) [reply ci-operator/config/openshift/release-tests/openshift-release-tests-master.yaml]: I changed it every 2h.

**Comment by wangke19** [comment]
/pj-rehearse periodic-ci-openshift-release-tests-master-qe-release-gate-test-arm64

**Comment by rioliu-rh** [comment]
working on PR https://github.com/openshift/release/pull/54794 to support configurable releases in job, currently for different arches we don't have test job definition for all of them

**Comment by rioliu-rh** [comment]
the code is merged, you can add new env var `OCP_RELEASES: 4.16` for your job, since you only have job definition for 4.16 release

File: ci-operator/jobs/openshift/release-tests/openshift-release-tests-master-periodics.yaml [hunk 3]
```diff
@@ -14,15 +14,6 @@ periodics:
     job-release: "4.16"
     pj-rehearse.openshift.io/can-be-rehearsed: "true"
   name: periodic-ci-openshift-release-tests-master-qe-release-gate-test
-  reporter_config:
```
Comment (rioliu-rh) [inline ci-operator/jobs/openshift/release-tests/openshift-release-tests-master-periodics.yaml:17]: this reporter config should not be removed

Comment (rioliu-rh) [reply ci-operator/jobs/openshift/release-tests/openshift-release-tests-master-periodics.yaml:17]: can you add it back to the prow job

File: ci-operator/config/openshift/release-tests/openshift-release-tests-master.yaml [hunk 4]
```diff
@@ -64,6 +64,14 @@ tests:
       OCP_RELEASES: 4.12 4.16
     test:
     - ref: release-qe-tests
+- as: qe-release-gate-test-arm64
+  cron: 0 */2 * * *
+  steps:
+    env:
+      OCP_ARCH: arm64
+      OCP_RELEASES: "4.16"
```
Comment (rioliu-rh) [inline ci-operator/config/openshift/release-tests/openshift-release-tests-master.yaml:72]: as I know, make update will reformat this file, i.e. remove double quote around 4.16

Comment (wangke19) [reply ci-operator/config/openshift/release-tests/openshift-release-tests-master.yaml]: Actually, it didn't, I Updated.

**Comment by rioliu-rh** [comment]
when PR https://github.com/openshift/release-tests/pull/236 is merged, we can run rehearsal job to verify whether arm job can be triggered

File: ci-operator/config/openshift/release-tests/openshift-release-tests-master.yaml (hunk 4 above)
Comment (wangke19) [reply ci-operator/config/openshift/release-tests/openshift-release-tests-master.yaml:72]: Have  to change back.
```
ERROR: The following errors were found:
diff -Naupr .//ci-operator/config/openshift/release-tests/openshift-release-tests-master.yaml /tmp/tmp.vbMR2s7oDU/ci-operator/config/openshift/release-tests/openshift-release-tests-master.yaml
--- .//ci-operator/config/openshift/release-tests/openshift-release-tests-master.yaml	2024-07-25 07:17:34.542187818 +0000
+++ /tmp/tmp.vbMR2s7oDU/ci-operator/config/openshift/release-tests/openshift-release-tests-master.yaml	2024-07-25 07:17:24.622267283 +0000
@@ -6[9](https://prow.ci.openshift.org/view/gs/test-platform-results/pr-logs/pull/openshift_release/54749/pull-ci-openshift-release-master-ci-operator-config-metadata/1816371438177751040#1:build-log.txt%3A9),7 +69,7 @@ tests:
   steps:
     env:
       OCP_ARCH: arm64
-      OCP_RELEASES: "4.16"
+      OCP_RELEASES: 4.16
     test:
     - ref: release-qe-tests
     - ```

**Comment by rioliu-rh** [comment]
/retitle OCPQE-24207 add release gate test job for arm64

**Comment by rioliu-rh** [comment]
/pj-rehearse periodic-ci-openshift-release-tests-master-qe-release-gate-test-arm64

**Comment by rioliu-rh** [comment]
from latest rehearsal job, found a bug in job controller. submit a PR to fix
https://github.com/openshift/release-tests/pull/238

**Pull request merged**